EdurecaInClassProject/src/app/components/login/login.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+  <div *ngIf="message" class="alert alert-danger m-3">{{message}}</div>
+  <div *ngIf="login" class="col-md-8 block user card card-body m-5">
+    <h2>Login</h2>
+    <form (ngSubmit)="onLogin()" [formGroup]="loginForm">
+        <div class="form-group">
+            <label>User Name</label>
+            <input type="text" required formControlName="user_name" minlength="6" class="form-control form-control-sm">
+            <div *ngIf="lf.user_name.errors && lf.user_name.errors.required && submitted  && lf.user_name.touched" class="text-danger">Username required</div>
+            <div *ngIf="lf.user_name.errors && lf.user_name.errors.minlength && submitted" class="text-danger">Username mrust be have atleast 8 character</div>
+        </div>
+        <div class="form-group">
+            <label>Password</label>
+            <input type="password" required formControlName="user_password" minlength="8" class="form-control form-control-sm">
+            <div *ngIf="lf.user_password.errors && lf.user_password.errors.required && lf.user_password.touched" class="text-danger">Paaword is required</div>
+            <div *ngIf="lf.user_password.errors && lf.user_password.errors.minlength && submitted" class="text-danger">Password should have minimum 8 digits</div>
+        </div>
+            <button type="submit" class="btn btn-primary">Login</button>
+            <button (click)="toggle()" class="btn btn-secondary m-3">Register</button>
+    </form>
+  </div>
+  <div *ngIf="register"  class="col-md-8 block user card card-body m-5">
+    <h2>Register</h2>
+    <form (ngSubmit)="onRegister()" [formGroup]="registerForm">
+      <div class="form-group">
+        <label>User Name</label>
+        <input type="text" required formControlName="user_name" minlength="6" class="form-control form-control-sm">
+        <div *ngIf="f.user_name.errors && f.user_name.errors.required && submitted  && f.user_name.touched" class="text-danger">Username required</div>
```

Comment:
Follow anyone naming convection at a time.
Personal suggestion : camelCase

Reply:
do I need to implement it now or is it okay if I follow that in the coming days?